src/Content/ResultServiceLocator.php

```diff
@@ -9,8 +9,10 @@
 
 namespace GisoStallenberg\Bundle\ResponseContentNegotiationBundle\Content;
 
+use GisoStallenberg\Bundle\ResponseContentNegotiationBundle\Event\NegotiatedResultDataEvent;
 use Negotiation\Accept;
 use Negotiation\Negotiator;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
```

Comment:
I'd rather have the interface from symfony/event-dispatcher-contracts which is a requirement of symfony/event-dispatcher 4.4 and 5.0

Reply:
Updated the interface 👍